api/src/validators/change-password.ts

```diff
@@ -0,0 +1,9 @@
+import z from "zod";
+
+export const changePasswordSchema = z.object({
+    currentPassword: z.string().min(8),
+    newPassword: z.string().min(8),
+    repeatPassword: z.string().min(8),
```

Comment:
É bom atualizar as mensagens de erro nos schemas assim a gente pode usar esse erro no front, caso necessário.

Coloca ali em cada message algo que ajude o usuário a entender o que ele fez de errado.

Testa e manda os prints das chamadas ao endpoint pra mim, por favor.

```suggestion
    currentPassword: z.string().min(8, { message: "" }),
    newPassword: z.string().min(8, { message: "" }),
    repeatPassword: z.string().min(8, { message: "" }),
```

Reply:
Testado:
![image](https://github.com/Somehow-I-Code/marquei/assets/133810168/15558e18-4e8e-426a-873e-80b5eee6e0e8)

Reply:
Tem como deixar claro qual das senhas? Exemplo "A repetição da sua senha tem que ter pelo menos 8 caracteres".

Reply:
Feito alteração no message.
A senha atual:
![image](https://github.com/Somehow-I-Code/marquei/assets/133810168/a6eedf90-43f8-48d0-9f64-1cc8e96f6fa7)
A senha nova:
![image](https://github.com/Somehow-I-Code/marquei/assets/133810168/1654e53d-7ebb-43fc-b321-ce573da13284)
A senha repetida:
![image](https://github.com/Somehow-I-Code/marquei/assets/133810168/2fa94050-f74f-4e0d-b12b-23c40be51b4b)
Ficaram bom?